src/main/java/nextstep/security/builder/AuthorizationConfigurer.java

```diff
@@ -0,0 +1,91 @@
+package nextstep.security.builder;
+
+import nextstep.security.access.AnyRequestMatcher;
+import nextstep.security.access.MvcRequestMatcher;
+import nextstep.security.access.RequestMatcher;
+import nextstep.security.access.RequestMatcherEntry;
+import nextstep.security.access.hierarchicalroles.NullRoleHierarchy;
+import nextstep.security.access.hierarchicalroles.RoleHierarchy;
+import nextstep.security.authorization.AuthenticatedAuthorizationManager;
+import nextstep.security.authorization.AuthorityAuthorizationManager;
+import nextstep.security.authorization.AuthorizationFilter;
+import nextstep.security.authorization.AuthorizationManager;
+import nextstep.security.authorization.DenyAllAuthorizationManager;
+import nextstep.security.authorization.PermitAllAuthorizationManager;
+import nextstep.security.authorization.RequestMatcherDelegatingAuthorizationManager;
+import org.springframework.http.HttpMethod;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AuthorizationConfigurer implements SecurityConfigurer<AuthorizationFilter, HttpSecurity> {
+
+    private final List<RequestMatcherEntry<AuthorizationManager>> mappings = new ArrayList<>();
+    private RoleHierarchy roleHierarchy = new NullRoleHierarchy();
+
+    public AuthorizationConfigurer(RoleHierarchy roleHierarchy) {
+        this.roleHierarchy = roleHierarchy;
+    }
+
+    @Override
+    public void init(HttpSecurity builder) {
+
+
+    }
+
+    @Override
+    public void configure(HttpSecurity builder) {
+        AuthorizationFilter authorizationFilter = new AuthorizationFilter(new RequestMatcherDelegatingAuthorizationManager(mappings));
+        builder.addFilter(authorizationFilter);
+    }
+
+
+    public AuthorizedEntry requestMatchers(String url) {
+        AuthorizedEntry authorizedEntry = new AuthorizedEntry(new MvcRequestMatcher(HttpMethod.GET, url));
+        return authorizedEntry;
+    }
+
+    public AuthorizedEntry anyRequest() {
+        return new AuthorizedEntry(AnyRequestMatcher.INSTANCE);
+    }
+
+    public class AuthorizedEntry {
+        private final RequestMatcher requestMatcher;
+
+        public AuthorizedEntry(RequestMatcher requestMatcher) {
+            this.requestMatcher = requestMatcher;
+        }
+
+        public AuthorizedEntry requestMatchers(String url) {
+            return new AuthorizedEntry(new MvcRequestMatcher(HttpMethod.GET, url));
```

Comment:
```suggestion
            return requestMatchers(HttpMethod.GET, url);
```

동일한 기능은 오버로딩을 활용해 하나의 메서드에서만 처리하면 유지보수성이 올라갈 것 같아요!